### DISCUSSION TITLE
Next 14 Aggressive Caching

### DISCUSSION THREAD

**Discussion by 0x4A756E65** [serwist/serwist#26] [Q&A]
Hi - I am using the basic serwist config, but am having issues with one of my main routes being aggressively served from cache. I have a route (`website.com/dashboard`) that shows many different metrics to my users. This is a server component page that fetches quite a bit of data initially, but I need that data the be fresh on each load for accuracy.

This is my current `sw.ts`

```
import { defaultCache } from "@serwist/next/browser";
import { PrecacheEntry } from "@serwist/precaching/dist";
import { installSerwist } from "@serwist/sw";

declare const self: ServiceWorkerGlobalScope & {
  // Change this attribute's name to your `injectionPoint`.
  __SW_MANIFEST: (PrecacheEntry | string)[] | undefined;
};

installSerwist({
  precacheEntries: self.__SW_MANIFEST,
  skipWaiting: true,
  clientsClaim: true,
  navigationPreload: true,
  runtimeCaching: defaultCache,
});
```

How can I fallback to cache, but prefer a network first/only approach?

**Comment by DuCanhGH** [comment]
[Shouldn't the routes be served with NetworkFirst by default?](https://github.com/serwist/serwist/blob/main/packages/next/src/index.browser.ts#L166-L199) That's really weird... Can you provide a reproduction? If you have used `@ducanh2912/next-pwa`, would you mind checking whether this issue is reproducible with it? Thanks!

**Reply by 0x4A756E65** [reply]
They may be and this is completely unrelated @DuCanhGH - thanks a lot for the link, I will do some more research
